### PR TITLE
OBPIH-2834 Rework expiration summary to align with expiration reports

### PR DESCRIPTION
**Pull request by bamoure**

### CODE REVIEW

File: grails-app/services/org/pih/warehouse/tableroapi/IndicatorDataService.groovy [hunk 1]
```diff
@@ -18,7 +18,8 @@ class IndicatorDataService {
     def dashboardService
 
     DataGraph getExpirationSummaryData(Location location, def params) {
-        Integer querySize = params.querySize ? params.querySize.toInteger() - 1 : 5
+        Integer querySize = params.querySize ? params.querySize.toInteger() + 1 : 7
```
Comment (jmiranda) [inline grails-app/services/org/pih/warehouse/tableroapi/IndicatorDataService.groovy:21]: Can you explain what's going on here? What does querySize represent?

Comment (bamoure) [reply grails-app/services/org/pih/warehouse/tableroapi/IndicatorDataService.groovy:21]: querySize is the date filter (1 month, 3 months, etc...). Usually it represents the index of the number we want to display in the results array, but in this case it's the number of months we want to show + 1 as the first one is "today", the expired items.
Meaning that if we select 3 months we actually want 4 points on the graph: today, +30 days, +60 days and +90 days

Comment (jmiranda) [reply grails-app/services/org/pih/warehouse/tableroapi/IndicatorDataService.groovy:21]: Ok thank you. Can you add a brief comment to the code to concisely explain that. If not just include this entire paragraph.

Comment (bamoure) [reply grails-app/services/org/pih/warehouse/tableroapi/IndicatorDataService.groovy:24]: Done

**Pull request merged**